lib/api/model/events.dart

```diff
@@ -31,6 +31,8 @@ sealed class Event {
           default: return UnexpectedEvent.fromJson(json);
         }
       case 'message': return MessageEvent.fromJson(json);
+      case 'update_message': return UpdateMessageEvent.fromJson(json);
+      case 'delete_message': return UpdateMessageEvent.fromJson(json);
```

Comment:
```suggestion
      case 'delete_message': return DeleteMessageEvent.fromJson(json);
```
:slightly_smiling_face:

Reply:
Eep! Thanks for the catch.